modules/document_repository/ajax/addCategory.php

```diff
@@ -61,7 +63,7 @@
     $www = $config->getSetting('www');
 
     $msg_data['newCategory'] = $www['url'] . 
-                               "/main.php?test_name=document_repository";
+                              $baseURL . "/document_repository/";
```

Comment:
I think   $www['url']  and $baseURL contain the same value. It will create a duplication of hostname in the URL.